src/commands/shared/ssh.ts

```diff
@@ -194,17 +195,10 @@ export const prepareRequest = async (
     throw "Server did not return a request id. Please contact support@p0.dev for assistance.";
   }
 
-  const { provisionedRequest, publicKey } = result;
+  const { provisionedRequest } = result;
 
   const sshProvider = SSH_PROVIDERS[provisionedRequest.permission.provider];
 
-  if (
-    sshProvider.validateSshKey &&
```

Comment:
we can no longer guarantee that a permission will include a valid ssh key because it can be uploaded at access time.